src/api/time/TimeContext.js

```diff
@@ -371,6 +373,17 @@ class TimeContext extends EventEmitter {
     }
   }
 
+  /**
+   * Get the timestamp of the current clock
+   * @returns {number} current timestamp of current clock regardless of mode
+   * @memberof module:openmct.TimeAPI#
+   * @method now
+   */
+
+  now() {
+    return this.timestamp;
```

Comment:
I don't think we should be separately tracking the current timestamp in the time context. I think the sole source of time should be the clock, so let's remove the `timestamp` member variable and have this function call the active clock itself.

Reply:
Good point!